### ISSUE TITLE
Allow setting custom resolution of X server

### ISSUE THREAD

**Issue by doorknob60** [corbinlc/GNURootDebian#36]
I have a 2013 Moto X with a screen resolution of 1280x720. By default, the X server starts with a 1024x768 resolution, meaning a small amount on the top and bottom are cut off and I either have to zoom or scroll. Also it does not fit the aspect ratio. There should be a way to set custom resolution to fit my screen and aspect ratio (if there is an option I could not find it), and otherwise fine tune it for things like Runescape to fit the window "snug".

**Comment by corbinlc** [comment]
I completely agree that this should be done.  I will update later when I am doing work on this.

**Comment by tcluri** [comment]
I have a 2014 Moto G, also with the same resolution. On rotate, is switching resolution possible? Would also like to have the custom resolution of X server, how or where do I find options to do this myself? 

Attached screen shot
![motogscreenshot](https://cloud.githubusercontent.com/assets/10505502/11895285/7fecf180-a5a3-11e5-8696-99ea5d89395d.png)

**Comment by wint3** [comment]
I have the same issue. I'm a beginner. Beginner as in "used a Linux Distribution for the first time ever" type. Great work, to say the least corbinlc. It would be easier to use if the X terminal had adjustable scale.

**Comment by eximmer** [comment]
I changed a geometry parameter from "1024x768" to "1280x768" in two files:
 - /support/startX
 - /usr/bin/tightvncserver

... and it worked for me.

Just search for 1024x768 and you'll find the right spots.

Thanks corbinlc for a fine piece of work.

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by eximmer** [comment]
Thanks for your interest Adam.  It's no longer an issue for me, I'm quite content to modify a couple files to get the resolution I need.

As always: Thanks to corbinlc for a fine piece of work.
